src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  min-height: 60px;
+  background-color: rgb(255, 255, 255);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
better to avoid height hardcoding, let the content define height of it's container